task_12/index.html

```diff
@@ -13,20 +13,22 @@
 </head>
 <body>
     <header>
-        <div class="time">
-            <span>
-                <i class="far fa-clock"></i>
-                <span id="time">00:00:00</span>
-            </span>
-        </div>
-        <div class="logo">
-            <a href="#">
-                <img src="img/pizza_logo.png" alt="pizza logo">
-            </a>
-        </div>
-        <div class="login">
-            <i class="fas fa-user"></i>
-            <a href="#">login/signup</a>
+        <div class="container">
+            <div class="time">
+                <span>
+                    <i class="far fa-clock"></i>
+                    <time datetime="" id="time">00:00:00</time>
```

Comment:
Add some fake date/time value to `datetime` attribute following the [specs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) or otherwise page may not pass html validation. You will change it later with JS.
Adding fake date now will give you hint what and how to change with JS in future.
`innerText` should not necessarily match the `datetime` attr value.

Please, do the same with all instances of `<time>`.

Reply:
Thanks for notice! Done